File: TaskManager/210042141_TaskManager/src/Logger.java
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class Logger {

    private Date timestamp;
    private String description;
    private List<Logger> logs = new ArrayList<>();

    private Logger(){}

    private void addLog(Logger log){
        logs.add(log);
    }
    private  void getLog(List<Logger> logs){
       for(Logger log : logs){
           System.out.println("Log: "+ log);
       }
    }

}

File: TaskManager/TaskManager/Services/Logger.cs
using System;
using System.Collections.Generic;
using TaskManager.Commands;
namespace TaskManager.Services;

public class Logger
{
    private readonly List<string> _logs = new List<string>();

    public void LogCommand(Command command)
    {
        var logEntry=$"{command.Timestamp}: {command.Description}";
        _logs.Add(logEntry);
    }
    public IEnumerable<string> GetLogs()
    {
        return _logs;
    }
}